CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
+## nf-core/mag version 1.0.0 -
 
-## 0.1.0 - 2018-05-22
-Initial release of nf-core/mag, created with the NGI-NFcookiecutter template: https://github.com/ewels/NGI-NFcookiecutter.
+Initial release of nf-core/mag, created with the [nf-core](http://nf-co.re/) template.
```

Comment:
Would maybe be nice to have a brief description of what the pipeline does at the first release. For later comparison in case new features are added.